commands/bmc.nicole

```diff
@@ -182,24 +180,30 @@ function cmd_datetime_show {
   format_date "${compact}" "${utc}" "$(date +%s)"
 }
 
+function call_netconfig() {
+  local FNAME=${FUNCNAME[1]##cmd_}
```

Comment:
looks like unusable variable.

Reply:
Why? It is used right in the next line. Here we strip `cmd_` from the start of the function name, in the next line we convert underscores into spaces.

Reply:
Yep, exactly. I've missed it.